Content.Server/Atmos/Components/BarotraumaComponent.cs

```diff
@@ -43,13 +46,18 @@ public void Update(float airPressure)
             switch (pressure)
             {
                 // Low pressure.
+                // TODO QUESTION So this bit of code behaves such that 'lowPressureMultiplier' only ever  makes the
```

Comment:
So this here isn't directly related to the damage refactor, I just saw it when changing the changeDamage() call. Is this supposed to behave like that?

Reply:
Do you... realize that you can just make `lowPressureMultiplier` higher than 1 to raise it, yes?
Most hardsuits in SS14 have their lowPressureMultiplier to 100.

Reply:
My question is if it should supportd a lowPressureMultiplier smaller than one, because as far as I can tell it doesn't. The lowPressureMultiplier can not make you more susceptible to low-pressure damage.

The case switch is effectively equal to a single if statement:
`if (pressure <= Atmospherics.WarningLowPressure) and (pressure*lowPressureMultiplier <= Atmospherics.WarningLowPressure)`
Or do I need to get more sleep?

Reply:
It... already supports a `lowPressureMultiplier` smaller than one, though?
This is behaving exactly as expected. If your suit gives you 100 `lowPressureMultiplier` and the environment's pressure is 5kPa, the "felt pressure" would be 500 kPa (and because "felt pressure" cannot hurt you if it's too high, you get no damage anymore)
If your suit gives you 0.5 `lowPressureMultiplier` and the environment's pressure is 50kPa, the "felt pressure" would be 0.5, therefore hurting you more.
There's no problem with this code, it's fine.

Reply:
And even if it is always greater than one, you could just make it a single inequality instead of two?

Reply:
Ah sorry, didn't refresh my page and didn't see your replay. The current code is this:

`
case var p when p <= Atmospherics.WarningLowPressure:
    pressure *= lowPressureMultiplier;
    if (pressure > Atmospherics.WarningLowPressure)
        goto default;
    elseDoDamage()
`

This is functionally equivalent to 

`
case var p when (p <= Atmospherics.WarningLowPressure) and (p*lowPressureMultiplier <= Atmospherics.WarningLowPressure):
    pressure *= lowPressureMultiplier;
    elseDoDamage()
`

Even with a lowPressureMultiplier below 1, you still need (p <= Atmospherics.WarningLowPressure) to be true?